pinot-spi/src/main/java/org/apache/pinot/spi/config/table/UpsertTTLConfig.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.spi.config.table;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import javax.annotation.Nullable;
+
+
+public class UpsertTTLConfig {
+
+  @JsonPropertyDescription("ttl time unit")
+  private String _ttlTimeUnit;
+  @JsonPropertyDescription("ttl time value")
+  private String _ttlTimeValue;
```

Comment:
1. can you include java docs for this class
2. It will be useful to include the supported time units in the `@JsonPropertyDescription` for `_ttlTimeUnit`

Reply:
@navina This PR is still in WIP status. Let me add the java docs and test coverage. 

thanks for code review!